addon/models/radar.js

```diff
@@ -275,14 +275,14 @@ export default class Radar {
 
     this._scrollHandler = () => {
       if (this.isTracking) {
-        this._nextScroll = run.scheduleOnce('sync', this, this.filterMovement);
+        run(() => this._nextScroll = run.scheduleOnce('sync', this, this.filterMovement));
```

Comment:
> Or I guess you could use run.debounce?
Ignore... I just read: https://github.com/runspired/ember-run-raf/blob/master/README.md#what-are-the-consequences